### ISSUE TITLE
[Docs]: Unclear documentation on Eventbridge pipe source when using self-managed Kafka

### ISSUE THREAD

**Issue by JPlanken** [hashicorp/terraform-provider-aws#37181]
### Documentation Link

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/pipes_pipe

### Description

The documentation is unclear about what to provide in the source field when creating an Event bridge pipe with a self-managed Kafka cluster as a source through Terraform.

The terraform configuration of Eventbrite pipes expects a source, typically an ARN, as seen in the documentation. 
Reference: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/pipes_pipe#source

However, this cluster doesn't have an ARN as it is not managed by AWS but by e.g. Confluent. 
What does a user fill in as a source in this case? This should be described more clearly.

Example code below:

```
resource "aws_pipes_pipe" "eventbridge_northbound" {
  depends_on = [aws_iam_role_policy.target]
  name       = "my_eventbridge_pipe"
  role_arn   = aws_iam_role.my_eventbridge_role.arn
  source     = ???
  target     = aws_lambda_function.eventbridge_target.arn

  source_parameters {
    self_managed_kafka_parameters {

      additional_bootstrap_servers = [data.aws_secretsmanager_secret.confluent_bootstrap_server]
      topic_name = var.topicname

      credentials {
        basic_auth = data.aws_secretsmanager_secret.confluent_auth_credentials
      }
    }
  }
}
```

### Would you like to implement a fix?

None

**Comment by CodeSinghh** [comment]
I have read this issue thoroughly and I want to solve this ✍️

**Comment by CodeSinghh** [comment]
📍I made the **necessary changes** and created the **PR**

**Comment by breathingdust** [comment]
Resolved by #37278
